src/api/features/features.api.js

```diff
@@ -118,11 +136,14 @@ async function identifyOnGeomAdminLayer({
                 imageDisplay: `${screenWidth},${screenHeight},96`,
                 geometryFormat: 'geojson',
                 geometryType: `esriGeometry${coordinate.length === 2 ? 'Point' : 'Envelope'}`,
-                limit: featureCount,
+                // there's a hard limit of 50 on our backend
+                // see https://api3.geo.admin.ch/services/sdiservices.html#id10
+                limit: Math.min(featureCount, 50),
```

Comment:
I would not do this here, we might change the backend in future to allow more than 50 and that will make the code hard to understand. Would be better to document the hard limit of 50 in the jsdoc of this function but that's all, if we update the limit to 100, then the backend will just fail and that's fine.

Reply:
I'll move this to an app config constant (in `src/config.js`) it should be then easier to adjust this value later on